sentry-types/src/protocol/envelope.rs

```diff
@@ -340,6 +384,18 @@ impl Envelope {
         Envelope::from_slice(&bytes)
     }
 
+    /// Creates a new Envelope from path without attempting to parse any items.
+    ///
+    /// The resulting Envelope's `items` will just be a binary blob.
+    pub fn from_path_raw<P: AsRef<Path>>(path: P) -> Result<Self, EnvelopeError> {
+        let mut bytes = std::fs::read(path).map_err(|_| EnvelopeError::UnexpectedEof)?;
+        let (header, offset) = Self::parse_header(&bytes)?;
```

Comment:
I would have to check what native is doing, but I believe it does not even parse the header. The header can potentially contain more than just the event id, so its best to keep that opaque as well.

Reply:
So should I just manually parse the event id?

Reply:
do we need it? its optional anyway

Reply:
Fair point. So should we just shove the whole file contents into the `items` in the raw case?

Reply:
Yes, I would say so.